gobblin-runtime/src/main/java/org/apache/gobblin/runtime/job_monitor/KafkaJobMonitor.java

```diff
@@ -100,6 +113,16 @@ protected void processMessage(MessageAndMetadata<byte[], byte[]> message) {
         } else if (parsedMessage instanceof Either.Right) {
           this.remmovedSpecs.inc();
           this.jobCatalog.remove(((Either.Right<JobSpec, URI>) parsedMessage).getRight());
+
+          // Refer FlowConfigsResources:delete to understand the pattern of flow URI
```

Comment:
This is actually from a JobSpec URI, so comment should refer to that and the Orchestrator and SpecProducer instead of the flow config.